toucan_connectors/mssql/mssql_connector.py

```diff
@@ -1,9 +1,17 @@
 from contextlib import suppress
+from logging import getLogger
 
-import pyodbc
 from pydantic import Field, StringConstraints, create_model
 from typing_extensions import Annotated
 
+try:
+    import pyodbc
+
+    CONNECTOR_OK = True
+except ImportError as exc:  # pragma: no cover
+    getLogger(__name__).warning(f"Missing dependencies for {__name__}: {exc}")
+    CONNECTOR_OK = False
+
 from toucan_connectors.common import pandas_read_sql
```

Comment:
on veut check pandas aussi non ?

Reply:
It wasn't imported before, so I don't think we need to. But you're right, we should guarantee pandas to be present. I'll add `pandas` to all connector-related extras